templates/Dockerfile-rpm.tmpl

```diff
@@ -1,6 +1,11 @@
 # vim:set ft=dockerfile:
 FROM %%os%%:%%release%%
 
+RUN [[ %%os%% != centos ]] || [[ %%release%% != 8 ]] || ( \
+    cd /etc/yum.repos.d/ && sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
+    && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.epel.cloud|g' /etc/yum.repos.d/CentOS-* \
+    )
+
```

Comment:
The current title of the PR and the change here seems unrelated. Can you elaborate?

You can consider moving this to another PR or updating the PR title, and description